proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+
+<lynX> If we let humanity live on in the mine field of an unregulated Internet, then we need to spend billions in educating the teachers so that they actually know what they are talking about. Probably easier to change the Internet so that it no longer is a mine field. Then *nobody* has to learn encryption tools, and *nobody* has to understand why they are necessary. The "alternative" would become the default as mandated by legislation (#ObCrypto).
+
+Social science
+
+Curriculum has to offer a focus media science that is up to date to accompany the practical skills. Lessons in Philosophy of technology where schools don't have philosophy classes.
+
+Art
+
+A focus on New Media has to be offered. E. G. teaching kids to cut videos to gain an understanding of how media are essentially montages. Elaborating on skills and methods in a creative way.
+
+PUBLIC SECTOR
+
+Institutions
+
+Public places and institutions are likewise able to foster and implement more democratic technologies. For instance libraries can set up their systems in a more open source secure way while at the same time spreading knowledge and best practices to its users. A good example for this is the American Library Freedom Project (https://libraryfreedomproject.org/)
+
+Public media, public advertisement
+
+Another place to push back the dominance of corporate power and abuse of digital technologies by other players is to oppose their all encompassing presence in media and advertising (TV, Internet, Public sphere). There have to be state- or EU funded campaigns advocating for the above mentioned solutions and their essential importance for a free, democratic and also economically more just society - think platform and surveillance capitalism where, corporations mine data to generate exorbitant surplusses, without paying taxes and drastically changing “the game” for unaware citizens respectively, in corporate talk, users.
+
+<lynX> Let's forbid unethical surveillance capitalism rather than de-facto accepting it by running (desperate) campaigns for the "alternatives."
+
+# 4 
+# 30.06.2018 Input DSC Vienna1 - 18-06-30
+
+With the rapidly rising importance of science and technology, the importance of their democratic control is also growing. Most of basic research is still financed by the public, either directly done by public institutions are in private institutions but partly funded via public grants. Only when it comes to making final products out of this, this is done mostly in the private realm. There, most patent applications are done. Thus, most of the costs for research are paid by the public while most gains are privatized. The current patent-system is extremely harmful. The most logical conclusion would be to abolish the patent-system all together: This way everyone can profit from the common knowledge.
```

Comment:
a strong claim - not sure that I would endorse in present form / without additional qualifications (just supporting open source doesn't go far enough toward a sustainable model for me). Joren's our IP expert, maybe he can take on this proposal? - kate

Reply:
Patents also have their useful aspects. I would suggest to work on societal taxes and dividends for redistribution, this way the gains the corps had through public research flows back to the citizen. See also 'European New Deal'. - lynx